cmd/gpu_plugin/gpu_plugin.go

```diff
@@ -195,9 +195,20 @@ func newDevicePlugin(sysfsDir, devfsDir string, options cliOptions) *devicePlugi
 	return dp
 }
 
+func isNil(r rm.ResourceManager) bool {
+	ret := true
+
+	switch r.(type) {
+	case rm.ResourceManager:
+		ret = false
+	}
+
+	return ret
+}
```

Comment:
`!isNil()` sounds a bit double negative.

Maybe something like:
```
func isValid(r rm.ResourceManager) bool {
    if r.(type) == rm.ResourceManager {
        return true
    }
    return false
}
...
if isValid(dp.resMan) {
```

Reply:
Thanks, a very good point! I'll change it.

Reply:
More descriptive, but longer function name would be `isValidType()`.

Reply:
can this go to a commit/PR of its own?

Reply:
It can. It was a drive-by-fix while I ran lint for the xelink sidecar code.

Reply:
I've seen the linter problem locally too but the CI does not catch it.